NDB.Covid19/NDB.Covid19.Droid/Resources/layout/welcome_what_is_new.xml

```diff
@@ -1,81 +1,72 @@
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
+	xmlns:android="http://schemas.android.com/apk/res/android"
+	xmlns:app="http://schemas.android.com/apk/res-auto"
+	android:layout_width="match_parent"
+	android:layout_height="match_parent"
+	android:paddingStart="@dimen/defaultHorizontalMargin"
+	android:paddingEnd="@dimen/defaultHorizontalMargin">
 
-    <TextView
-        android:id="@+id/welcome_what_is_new_title"
-        style="@style/HeaderText"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:layout_alignParentTop="true"
-        android:paddingTop="44dp"
-        android:paddingHorizontal="25dp"
-        android:text="Nyt smitte|stop"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+	<TextView
+		android:id="@+id/welcome_what_is_new_title"
+		style="@style/HeaderText"
+		android:layout_width="0dp"
+		android:layout_height="wrap_content"
+		android:layout_alignParentTop="true"
+		android:paddingTop="@dimen/defaultHorizontalMargin"
+		android:text="What is new in Smittestopp"
+		app:layout_constraintEnd_toEndOf="parent"
+		app:layout_constraintStart_toStartOf="parent"
+		app:layout_constraintTop_toTopOf="parent" />
 
-    <ScrollView
-        android:id="@+id/scroll_view"
-        style="@style/ScrollScreen"
-        android:layout_width="match_parent"
-        android:layout_height="0dp"
-        app:layout_constraintBottom_toTopOf="@id/ok_button"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/welcome_what_is_new_title"
-        android:fillViewport="true">
+	<ScrollView
+		android:id="@+id/scroll_view"
+		style="@style/ScrollScreen"
+		android:layout_width="match_parent"
+		android:layout_height="0dp"
+		app:layout_constraintEnd_toEndOf="parent"
+		app:layout_constraintStart_toStartOf="parent"
+		app:layout_constraintTop_toBottomOf="@id/welcome_what_is_new_title"
+		android:fillViewport="true">
 
-        <LinearLayout
-            style="@style/CheckmarkContainer"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:orientation="vertical">
+		<LinearLayout
+			style="@style/CheckmarkContainer"
+			android:layout_width="match_parent"
+			android:layout_height="wrap_content"
+			android:orientation="vertical">
 
-            <include
-                android:id="@+id/bullet_one"
-                layout="@layout/welcome_bullet"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginBottom="18dp" />
+			<include
+				android:id="@+id/bullet_one"
+				layout="@layout/welcome_bullet"
+				android:layout_width="match_parent"
+				android:layout_height="wrap_content"/>
```

Comment:
no spacing between bullets?

Reply:
we don't have spacing between bullets in the welcome pages - that's why I removed it :D